### PR TITLE
[stable10] Test for creating tags for files

### PR DESCRIPTION
**Pull request by skshetry**
## Description

Backport for #32883

### CODE REVIEW

File: tests/acceptance/features/webUITags/createTags.feature [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  As a user
+  I want to create tags for the files/folders
+  So that I can find them easily
+
+  Background:
+    Given these users have been created:
+      | username | password  | displayname | email        |
+      | user1    | %regular% | User One    | u1@oc.com.np |
+    And the user has browsed to the login page
+    And the user has logged in with username "user1" and password "qwerty" using the webUI
```
Comment (phil-davis) [inline tests/acceptance/features/webUITags/createTags.feature]: I expect that this login will fail.
The password neeeds to be ``%regular%``

**Review by phil-davis — CHANGES_REQUESTED** [review]
See comments in master PR. It needs ``.drone.yml`` entries.

**Review by phil-davis — APPROVED** [review]

**Comment by phil-davis** [comment]
@skshetry there was no drone result reported. Anyway, because of the code style check rules added to core yesterday, it will be good for you to to rebase and run CI again.

**Comment by phil-davis** [comment]
@ownclouders rebase

**Comment by ownclouders** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being rebased automatically. Please **DO NOT** push while rebase is in progress or your changes would be lost permanently :warning:

**Comment by ownclouders** [comment]
Automated rebase with [GitMate.io](https://gitmate.io) was successful! :tada:

**Review by phil-davis — APPROVED** [review]

**Comment by phil-davis** [comment]
@ownclouders rebase

**Comment by ownclouders** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being rebased automatically. Please **DO NOT** push while rebase is in progress or your changes would be lost permanently :warning:

**Comment by ownclouders** [comment]
Automated rebase with [GitMate.io](https://gitmate.io) was successful! :tada:

**Comment by phil-davis** [comment]
@skshetry one of the tags tests failed???

**Comment by phil-davis** [comment]
@ownclouders rebase

**Comment by ownclouders** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being rebased automatically. Please **DO NOT** push while rebase is in progress or your changes would be lost permanently :warning:

**Comment by ownclouders** [comment]
Automated rebase with [GitMate.io](https://gitmate.io) was successful! :tada:

**Pull request merged**